canonical-web-and-design/ubuntu.com
# 202/account users/pagination

## Done

- Add pagination to Account users page results
  - handle response errors in requestAccountUsers
  - move cypress tests to `tests/cypress/integration`

## QA

- Go to advantage/users and login
- Add more users if you can't see pagination (pagination is enabled from 10 results +)
- Paginate through results and make sure nothing breaks


## Issue / Card

https://github.com/canonical-web-and-design/commercial-squad/issues/202

## Screenshots
